.gitignore

```diff
@@ -114,6 +114,7 @@ env/
 
 ### Configuration files ###
 .env
+docker.env
```

Comment:
What's this file for?

Reply:
It was for secrets...I wonder if I added it then changed plan...I'll try to remove